### PR TITLE
SNOW-921965: document update and correction

### PR DESCRIPTION
**Pull request by sfc-gh-ext-simba-hx**
teamwork issue 667

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -127,9 +122,9 @@ To build the PHP driver for Windows:
 
    where:
 
-   - :code:`<arch>` is your CPU architecture (:code:`x64` or :code:`x86`).
+   - :code:`<arch>` is your CPU architecture (Currently :code:`x64` is the only supported one).
```
Comment (sfc-gh-mheavin) [inline README.rst:125]: ... architecture. Currently, the driver only supports :code:`x64`.

File: README.rst [hunk 2]
```diff
@@ -127,9 +122,9 @@ To build the PHP driver for Windows:
...
    - :code:`<build>` is the type of binary that you want to build (:code:`Release` or :code:`Debug`).
-   - :code:`<visual studio version>` is the version of Visual Studio that you are using (:code:`VS16`).
+   - :code:`<visual studio version>` is the version of Visual Studio that you are using (Currently :code:`VS16` is the only supported one).
```
Comment (sfc-gh-mheavin) [inline README.rst:127]: (see above comment)

File: README.rst [hunk 3]
```diff
@@ -461,6 +488,56 @@ and verify no error in the output:
 
      ERROR SUMMARY: 0 errors from 0 contexts ...
 
+Running Tests For the PHP PDO Driver on Windows
+================================================================================
+
+In order to run the test scripts, you must have jq installed.
```
Comment (sfc-gh-mheavin) [inline README.rst:494]: To run the test scripts, jq must be installed.   [Should jq be in code font?]

File: README.rst [hunk 4]
```diff
@@ -461,6 +488,56 @@ and verify no error in the output:
...
+
+Prepare for Test
+----------------------------------------------------------------------
+
```
Comment (sfc-gh-mheavin) [inline README.rst:498]: These should be a numbered list

File: README.rst [hunk 5]
```diff
@@ -461,6 +488,56 @@ and verify no error in the output:
...
+Call :code:`env.bat` script to set the test connection parameters.
+
+.. code-block:: batch
+
+    .\scripts\env.bat
+
+
+Run Tests
+----------------------------------------------------------------------
+
```
Comment (sfc-gh-mheavin) [inline README.rst:530]: To run the tests, do the following:

**Review by sfc-gh-mheavin — COMMENTED** [review]
Some comments

File: README.rst [hunk 6]
```diff
@@ -426,39 +426,41 @@ Example usage:
 Running Tests For the PHP PDO Driver on Linux and macOS
 ================================================================================
 
-In order to run the test scripts, you must have jq installed.
+In order to run the test scripts, you must have :code:`jq` installed.
 
 Prepare for Test
```
Comment (sfc-gh-mheavin) [inline README.rst:431]: Prepare Tests

File: README.rst [hunk 7]
```diff
@@ -491,43 +493,45 @@ and verify no error in the output:
 Running Tests For the PHP PDO Driver on Windows
 ================================================================================
 
-In order to run the test scripts, you must have jq installed.
+In order to run the test scripts, you must have :code:`jq` installed.
 
 Prepare for Test
```
Comment (sfc-gh-mheavin) [inline README.rst:498]: Prepare Tests

**Review by sfc-gh-mheavin — APPROVED** [review]
A couple more minor suggestions. Approved.

File: README.rst [hunk 8]
```diff
@@ -398,40 +423,44 @@ Example usage:
...
+               "SNOWFLAKE_TEST_ACCOUNT":   "<your_account>",
+               "SNOWFLAKE_TEST_WAREHOUSE": "<your_warehouse>",
+               "SNOWFLAKE_TEST_DATABASE":  "<your_database>",
+               "SNOWFLAKE_TEST_SCHEMA":    "<your_schema>",
+               "SNOWFLAKE_TEST_ROLE":      "<your_role>"
+           }
+       }
 
-Call :code:`env.sh` script to set the test connection parameters in the environment variables.
+#. Set the workfolder to :code:`pdo_snowflake` repository. .e.g Call :code:`cd pdo_snowflake`.
```
Comment (sfc-gh-dprzybysz) [inline README.rst:450]: let's replace `. .e.g` with `, e.g.` (include dot after g)

File: README.rst [hunk 9]
```diff
@@ -461,6 +490,58 @@ and verify no error in the output:
...
+               "SNOWFLAKE_TEST_PASSWORD":  "<your_password>",
+               "SNOWFLAKE_TEST_ACCOUNT":   "<your_account>",
+               "SNOWFLAKE_TEST_WAREHOUSE": "<your_warehouse>",
+               "SNOWFLAKE_TEST_DATABASE":  "<your_database>",
+               "SNOWFLAKE_TEST_SCHEMA":    "<your_schema>",
+               "SNOWFLAKE_TEST_ROLE":      "<your_role>"
+           }
+       }
+
+#. Set the workfolder to :code:`pdo_snowflake` repository. .e.g Call :code:`cd pdo_snowflake`.
```
Comment (sfc-gh-dprzybysz) [inline README.rst:517]: let's replace `. .e.g` with `, e.g.` (include dot after g)

**Review by sfc-gh-dprzybysz — APPROVED** [review]

**Pull request merged**